app/controllers/admin/editionable_social_media_accounts_controller.rb

```diff
@@ -5,10 +5,10 @@ class Admin::EditionableSocialMediaAccountsController < Admin::BaseController
   def confirm_destroy; end
 
   def create
-    social_media_account = SocialMediaAccount.create(social_media_account_params)
```

Comment:
Had to change this a bit because Rubocop didn't like the `create`

Reply:
Thanks for this explanation, love a bit of Rubocop weirdness 🙃